web/src/main/java/org/cbioportal/web/config/CustomObjectMapper.java

```diff
@@ -39,68 +39,16 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.SerializationFeature;
 
-import org.cbioportal.model.CancerStudy;
-import org.cbioportal.model.ClinicalAttribute;
-import org.cbioportal.model.ClinicalAttributeCount;
-import org.cbioportal.model.ClinicalData;
-import org.cbioportal.model.ClinicalDataCount;
-import org.cbioportal.model.ClinicalEvent;
-import org.cbioportal.model.ClinicalEventData;
-import org.cbioportal.model.DataAccessToken;
-import org.cbioportal.model.CopyNumberSeg;
-import org.cbioportal.model.Gene;
-import org.cbioportal.model.GenePanel;
-import org.cbioportal.model.GenePanelToGene;
-import org.cbioportal.model.Geneset;
-import org.cbioportal.model.GenesetCorrelation;
-import org.cbioportal.model.GenesetMolecularData;
-import org.cbioportal.model.Gistic;
-import org.cbioportal.model.GisticToGene;
-import org.cbioportal.model.MolecularProfile;
-import org.cbioportal.model.MutSig;
-import org.cbioportal.model.Mutation;
-import org.cbioportal.model.MutationSpectrum;
-import org.cbioportal.model.Patient;
-import org.cbioportal.model.ResourceDefinition;
-import org.cbioportal.model.Sample;
-import org.cbioportal.model.SampleList;
+import org.cbioportal.model.*;
```

Comment:
Better to restore the old import format.

Reply:
Fixed in [e741f34](https://github.com/BasLee/cbioportal/pull/1/commits/e741f34f8a023df66b841a406d323e650cdf2308)